src/Datadog.Trace/ServiceFabric/Stubs.cs

```diff
@@ -0,0 +1,149 @@
+        /// For internal use only.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        IServiceRemotingResponseMessageHeader GetHeader();
+    }
+
+    /// <summary>
+    /// For internal use only.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public interface IServiceRemotingRequestMessageHeader
+    {
+        /// <summary>
+        /// Gets the method id. For internal use only.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        int MethodId { get; }
+
+        /// <summary>
+        /// Gets the method identifier. For internal use only.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        int InterfaceId { get; }
+
+        /// <summary>
+        /// Gets the invocation identifier. For internal use only.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        string? InvocationId { get; }
+
+        /// <summary>
+        /// Gets the method name. For internal use only.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        string? MethodName { get; }
+
+        /// <summary>
+        /// For internal use only.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        void AddHeader(string headerName, byte[] headerValue);
+
+        /// <summary>
+        /// For internal use only.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        bool TryGetHeaderValue(string headerName, out byte[]? headerValue);
+    }
+
+    /// <summary>
+    /// For internal use only.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public interface IServiceRemotingRequestMessage
+    {
+        /// <summary>
+        /// For internal use only.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        IServiceRemotingRequestMessageHeader GetHeader();
+    }
+
+    /// <summary>
+    /// For internal use only.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public interface IServiceRemotingRequestEventArgs
+    {
+        /// <summary>
+        /// Gets the request message. For internal use only.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IServiceRemotingRequestMessage? Request { get; }
+
+        /// <summary>
+        /// Gets the service URI. For internal use only.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Uri? ServiceUri { get; }
+
+        /// <summary>
+        /// Gets the method name. For internal use only.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string? MethodName { get; }
+    }
+
+    /// <summary>
+    /// For internal use only.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public interface IServiceRemotingResponseEventArgs
```

Comment:
And do we need this type as well?

Reply:
Thanks. Another left over from previous iterations. Removed in 9fca6b24.